int8 ec_id      # ethercat板子的索引
int8 slave_id   # ethercat板子的子板的索引
